packages/vue-components/src/Navbar.vue

```diff
@@ -132,9 +132,10 @@
         const navLis = Array.from(this.$el.querySelector('.navbar-nav').children);
         // attempt an exact match first
         for (const li of navLis) {
+          const standardLinks = [li];
```

Comment:
hmm what's this for? 😮 

tried removing this and it worked fine still

Reply:
Oh this is for cases where the navbar does not use the `li` tags. 
For example:
```
<header>
  <navbar type="dark">
    <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand">Your Logo</a>
    <a href="{{baseUrl}}/contents/topic1.html" class="nav-link">Topic 1</a>
    <a href="{{baseUrl}}/contents/topic2.html" class="nav-link">Topic 2</a>
    <a href="{{baseUrl}}/contents/topic3a.html" class="nav-link">Topic 3a</a>
    <a href="{{baseUrl}}/contents/topic3b.html" class="nav-link">Topic 3b</a>
  </navbar>
</header>
```
In this case, without this line, the highlighting would not work properly as `navLinks` only looks for anchortags within `li`.

Reply:
thanks for the explanation; nice catch 👍